code/game/turfs/simulated/floor/chasm.dm

```diff
@@ -152,8 +152,18 @@
 			L.adjustBruteLoss(30)
 	falling_atoms -= AM
 
+/turf/simulated/floor/chasm/straight_down
+	var/obj/effect/abstract/chasm_storage/storage
```

Comment:
So, what happens if the turf type is changed somehow? Will the stuff in the storage just get deleted forever?

Reply:
Storage is not deleted when the chasm is changed, it basically acts like part of the turfs contents

Reply:
Well, in that case, what happens to it? Does it just get stuck in turf contents forever?

Reply:
Until the turf is converted back to a chasm, or a changeling revives in it, yes